dist/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Grafana
```

Comment:
What do we do about this line? 🤔 

I'd remove the year for sure, not sure about the org

Reply:
Seems the year is part of it according to https://opensource.org/licenses/MIT. I'd another line with the two of us or maybe only you?

Reply:
You're right, let's leave the year. Add both ;)